doc/howto/write_a_view/index.md

```diff
@@ -0,0 +1,429 @@
+# How to write a view {#howto_write_a_view}
+
+[TOC]
+
+This HowTo documents how to write a view using the standard library and some helpers from SeqAn3.
+
+\tutorial_head{Difficult, 120 min, \ref tutorial_concepts\,\ref tutorial_ranges, }
```

Comment:
```suggestion
\tutorial_head{Hard, 120 min, \ref tutorial_concepts\,\ref tutorial_ranges, }
```
The difficulty is difficult 🤓

Reply:
But "hard" is more colloquial,

Reply:
High then?

Reply:
Then we need to change the others to "low" and "medium". I don't have a problem with "Difficulty: difficult" though.